### PR TITLE
Progressbar

### PR DESCRIPTION
**Pull request by juusechec**
Adding progress bar to dialog generate, export and import

### CODE REVIEW

**Comment by juusechec** [comment]
I've update with master branch to show that this branch hasn't conflicts. We waiting for observations.

File: projectgenerator/libili2db/ili2dbutils.py [hunk 1]
```diff
@@ -77,3 +77,13 @@ def get_ili2db_bin(tool_name, stdout, stderr):
             return None
 
     return ili2db_file
+
+def get_all_modeldir_in_path(path, lambdafunction=None):
```
Comment (m-kuhn) [inline projectgenerator/libili2db/ili2dbutils.py]: Is this related to the progress bar?
